src/transforms/filter.js

```diff
@@ -8,211 +8,269 @@
 
 'use strict';
 
-var isNumeric = require('fast-isnumeric');
-
-// var Lib = require('@src/lib');
 var Lib = require('../lib');
+var axisIds = require('../plots/cartesian/axis_ids');
 
-/* eslint no-unused-vars: 0*/
-
-// so that Plotly.register knows what to do with it
 exports.moduleType = 'transform';
 
-// determines to link between transform type and transform module
 exports.name = 'filter';
 
-// ... as trace attributes
 exports.attributes = {
+    active: {
```

Comment:
I added an `active` attribute similar to the `active` in the groupby transform.

**BUT**, if no one objects, I'll change `active` -> `visible` for consistency with our other components. Moreover, `active` is already used in `layout.updatemenus` where it an `'integer'` attribute, having both a boolean and an integer `active` attribute in our schema would be pretty confusing.